src/app/templates/activity/activity-item/hooks/use-activity-item-view-model.tsx

```diff
@@ -69,6 +71,14 @@ export const useActivityItemViewModel = (activity: DisplayableActivity) => {
       return <BakerLogo bakerAddress={actor.address} />;
     }
 
+    if (is3Route) {
```

Comment:
I would instead create a dictionary of logos for interaction subtypes like this and use it here to make adding other dApps easier